app/workers/fetch_public_posts.rb

```diff
@@ -4,7 +4,7 @@
 
 module Workers
   class FetchPublicPosts < Base
-    sidekiq_options queue: :http_service
+    sidekiq_options queue: :high
```

Comment:
I'd downrate this to `medium` or even `low`, having some public posts when viewing a profile the first time is nice, but not mandatory.